roles/apply-config-update/tasks/main.yaml

```diff
@@ -5,4 +5,7 @@
 - name: Apply beta-3 changes
   import_tasks: beta-3.yaml
 
+- name: Apply beta-4 changes
```

Comment:
the next release is 3, please use `beta-3.yaml` to make changes for `beta-3`.

Reply:
understand, then I'll put the content of `beta-4.yaml` into the `beta-3.yaml`

Reply:
`beta-4.yaml` is removed and content of the `beta-4.yaml` is moved into `beta-3.yaml` file